src/views/soknad/Soknad.tsx

```diff
@@ -72,7 +74,7 @@ export function Soknad(props: IProps) {
     <main>
       {/*<ProgressBar currentStep={currentSectionIndex + 1} totalSteps={sectionsCount} />*/}
 
-      {isFirstSection && props.personalia && (
+      {showPersonalia && props.personalia && (
```

Comment:
Kunne `props.personalia` vært en dele av showPersonalia const?

Reply:
Det var det først, men da begynte typescript å klage på at props.personalia også kunne være `null` i Personalia-komponenten. Den var ikke smart nok til å forstå at showPersonalia også sjekket det for deg. :/ Dermed separerte jeg den i to sjekker.

Reply:
ahaa, 👍